odml/addons.py

```diff
@@ -0,0 +1,23 @@
+import odml.tools.xmlparser as X
+import odml.tools.jsonparser as Y 
+
```

Comment:
Why import them as `X` and `Y`? Why not something meaningful like `xml` and `json`?